9184_cpp_wodnjs.cpp

```diff
@@ -0,0 +1,60 @@
+#include<iostream>
+
+using namespace std;
+int d[101][101][101];
+bool vis[101][101][101];
+int get_w(int a, int b, int c );
+int main() {
+	for (int i = 0; i < 101; i++)
+	{
+		for (int j = 0; j < 101; j++)
+		{
+			for (int k = 0; k < 101; k++)
+			{
+				if (i <= 50 || j <= 50 || k <= 50)
+				{
+					d[i][j][k] = 1;
+					vis[i][j][k] = true;
+				}
+			}
+		}
+	}
+	for (int i = 0; i < 101; i++)
+	{
+		for (int j = 0; j < 101; j++)
+		{
+			for (int k = 0; k < 101; k++)
+			{
+				get_w(i, j, k);
+			}
+		}
+	}
+	while (1) {
+		int a, b, c;
+		cin >> a >> b >> c;
+		if (a == -1 && b == -1 && c == -1)
+			return 0;
+		cout << "w(" << a << ", " << b << ", " << c << ") = " << d[a+50][b+50][c+50] << '\n';
+	}
+	return 0;
+}
+int get_w(int a, int b, int c) {
+
```

Comment:
이곳에 음수일때 예외처리하면 a,b,c에 특정 값을 더해서 index를 조정하는 일을 하지 않아도 됩니다. 아래있는 코드도 전부 고쳐주세요.
```suggestion
      if(a<=0 || b<=0 || c<=0)
          return 1;
```